.dockerignore

```diff
@@ -1,9 +1,6 @@
-node_modules
-npm-debug.log
+node_modules/
+dist/
 Dockerfile
 .dockerignore
-.git
-.gitignore
-README.md
-LICENSE
-.vscode
\ No newline at end of file
+.vscode/
+.env
```

Comment:
Bez .env działa?

Reply:
nie możesz wrzucać `.env` do obrazu dockera, bo to są wrażliwe dane, ENV podajesz w komendzie `docker run`